roles/sro_deploy_art_bundle/files/builder_script.yml

```diff
@@ -0,0 +1,46 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  labels:
+    app: ci-artifacts
+  name: ci-artifacts-script
+  namespace: ci-artifacts
+data:
+  run_index_builder.sh: |
+    #! /bin/bash
+    set -x
+    set -o nounset
+
+    IMAGE_NAME="brew.registry.redhat.io/rh-osbs/openshift-ose-special-resource-operator-bundle"
+
+    BUNDLES="/tmp/bundles"
+    podman search --authfile /var/lib/kubelet/config.json \
+      ${IMAGE_NAME} \
+      --list-tags --limit 500 | sort -r > ${BUNDLES}
+
+    git clone -b ${GIT_COMMIT} ${GIT_REPO} /tmp/special-resource-operator
+    cd /tmp/special-resource-operator
+    COMMITS=`git log --pretty=format:"%h"`
+
+    for commit in ${COMMITS}; do
+      match=`egrep -m 1 ".+\.${commit}\..+" ${BUNDLES}`
+      if [[ -n ${match} ]]; then
+        break
+      fi
+    done
+    if [[ -z ${match} ]]; then
+      echo "No bundles found in registry"
+      exit 1
+    fi
+    BUNDLE=`echo ${match} | awk '{print $1":"$2}'`
+
+    echo "BUNDLE=${BUNDLE}"
+    echo "CATALOG_INDEX=${CATALOG_INDEX}"
+    echo "COMMIT=${commit}"
+
+    REGISTRY_AUTH_FILE=/var/lib/kubelet/config.json /usr/local/go/bin/opm index add --bundles ${BUNDLE} --tag ${CATALOG_INDEX} -c podman
+
+    AUTH="--tls-verify=false --authfile /tmp/.dockercfg"
+    cp /var/run/secrets/openshift.io/push/.dockercfg /tmp
+    (echo "{ \"auths\": " ; cat /var/run/secrets/openshift.io/push/.dockercfg ; echo "}") > /tmp/.dockercfg
+    podman push $AUTH $CATALOG_INDEX
```

Comment:
The bundle I get from ART is already built, so everything that is left is to include it in an index and then spawn a pod from it. I chose not to use quay because I didnt want the index to leave the cluster, so I turned to the local image registry. This required some RBAC policies for the openshift-marketplace namespace and the new catalog pod service account. Had no trouble apart from that, which prevents the pod from starting.